NOTICE

```diff
@@ -0,0 +1,50 @@
+  Copyright 2017, 2018 SPDX
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+ 
+ The following open source components are included in the source.
+ See the requirements.txt file for all 3rd party and open source components included as dependencies.
+
+ This package includes CodeMirror (https://codemirror.net/)
+ CodeMirror is licensed under MIT License
+ 
```

Comment:
If you can find that actual MIT license with the CodeMirror copyright and paste it here, that would be better than the reference to the MIT license - but this is OK as well